src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,69 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+    public static final int LOTTO_TICKET_NUMBER = 6;
+    public static final int LOTTO_TICKET_PRICE = 1_000;
+    private final List<LottoNumber> list = new ArrayList<>();
+
+    public LottoTicket(List<LottoNumber> subList) {
+        validateSize(subList);
+        validateDuplication(subList);
+        this.list.addAll(subList);
```

Comment:
addAll이 아닌 아래와 같이 즉시 할당하는 형태로 구현해 불변을 유지하면 어떨까요?

```suggestion
        this.list = subList;
```

Reply:
수정했습니다! 생각없이 사용하던 부분이었는데 관련해서 좀더 공부해보겠습니다.